### PR TITLE
Generate connection profiles

### PR DESCRIPTION
**Pull request by gzhk**
closes: #179

### CODE REVIEW

File: e2e/performTests.ts [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
   files.forEach((f) => {
     it(`should create proper ${f} from ${config}`, () => {
       const content = commands.getFileContent(`${commands.relativeRoot}/${f}`);
       const cleaned = content
         .replace(/FABLO_BUILD=(.*?)(\n|$)/g, "FABLO_BUILD=<date with git hash>\n")
         .replace(/FABLO_CONFIG=(.*?)(\n|$)/g, "FABLO_CONFIG=<absolute path>\n")
         .replace(/CHAINCODES_BASE_DIR=(.*?)(\n|$)/g, "CHAINCODES_BASE_DIR=<absolute path>\n")
-        .replace(/COMPOSE_PROJECT_NAME=(.*?)(\n|$)/g, "COMPOSE_PROJECT_NAME=<name with timestamp>\n");
+        .replace(/COMPOSE_PROJECT_NAME=(.*?)(\n|$)/g, "COMPOSE_PROJECT_NAME=<name with timestamp>\n")
+        .replace(new RegExp(rootPath, "g"), "<absolute path>");
```
Comment (dzikowski) [inline e2e/performTests.ts:19]: sprytnie z tym rootPath. Zadziała to może też na `FABLO_CONFIG` i `CHAINCODES_BASE_DIR`? bo jeśli tak, to bym tamte replacy proponował wywalić

druga rzecz - schowałbym ten rootPath tutaj, tzn nie jako parameter, tylko tutaj resolvował `const rootPath = resolve(__dirname + "/../");`

File: src/types/ConnectionProfile.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import { NetworkSettings, OrgConfig, PeerConfig } from "./FabloConfigExtended";
```
Comment (dzikowski) [inline src/types/ConnectionProfile.ts:1]: fajnie, że w TS 👏

**Review by dzikowski — APPROVED** [review]

**Review by Hejwo — APPROVED** [review]

File: e2e/performTests.ts (hunk 1 above)
Comment (gzhk) [reply e2e/performTests.ts:19]: To nie zadziałało, tam były jakieś inne wartości.

**Comment by manjinder98** [comment]
Hi guys!
I noticed you implemented this feature for generating connection profiles but I am not able to find them in fabric-config as specified in the README file.
Screenshot 2021-12-27 at 12 16 01

**Comment by dzikowski** [comment]
Hi, it is just not released yet. We plan to make a new release in the next few days, so you can just wait, or you can build Fablo from the source code.

If you want to use Fablo from the source code:
1. Download the source code (`main` branch)
2. Execute `fablo-build.sh`
3. Use version `0.4.0` in your Fablo config in `$schema` attribubute
4. Use `fablo.sh` script from source code instead of `./fablo`

**Comment by manjinder98** [comment]
> Hi, it is just not released yet. We plan to make a new release in the next few days, so you can just wait, or you can build Fablo from the source code.
> 
> If you want to use Fablo from the source code:
> 
> 1. Download the source code (`main` branch)
> 2. Execute `fablo-build.sh`
> 3. Use version `0.4.0` in your Fablo config in `$schema` attribubute
> 4. Use `fablo.sh` script from source code instead of `./fablo`

That's perfect! Thanks very much!

**Pull request merged**